.github/workflows/annotate.yml

```diff
@@ -0,0 +1,58 @@
+name: Annotate checks
+
+on:
+  workflow_run:
+    workflows: ["ci", "docs"]
+    types:
+      - completed
+
+jobs:
+  annotate:
+    if: github.repository_owner == 'trinodb' && github.event.workflow_run.event == 'pull_request'
```

Comment:
Any reason why we don't want to annotate failures on merges to master?

Reply:
This is a big oversight on my side, thanks!